src/components/UI/Card/CartCards/CartCards.jsx

```diff
@@ -1,88 +1,137 @@
+        quantity
+    } = props;
+    const {cartState, setCartState} = useCart();
+    const {WishListState, setWishListState} = useWishList();
+
+    const RemoveItemsFromCartHandler = async (id) => {
+        try {
             console.log(id);
-            await axios.delete(`/api/user/cart/${id}`,
-                    { headers:{authorization:localStorage.getItem("feetz")}})
-                        .then((res)=>{
-                            console.log(res);
-                            if(res.status===200){
-                                setCartState(res.data.cart);
-                            }
-                        })
-                        .catch((error)=>{
-                            console.log(error.message);
-                        });
-        }
-        catch(err){
-            console.log("error ",err.message);
+            await axios.delete(`/api/user/cart/${id}`, {
+                headers: {
+                    authorization: localStorage.getItem("feetz")
+                }
+            }).then((res) => {
+                console.log(res);
+                if (res.status === 200) {
+                    setCartState(res.data.cart);
+                }
+            }).catch((error) => {
+                console.log(error.message);
+            });
+        } catch (err) {
+            console.log("error ", err.message);
         }
     }
 
-    const addToWishlistHandler = async (item)=>{
-        try{
+    const addToWishlistHandler = async (item) => {
+        try {
             console.log(item);
-            await axios.post("/api/user/wishlist",
-                    { "product":item},
-                    { headers:{authorization:localStorage.getItem("feetz")}})
-                        .then((res)=>{
-                            console.log(res);
-                            if(res.status===201){
-                                setWishListState(res.data.wishlist);
-                            }
-                        })
-                        .catch((error)=>{
-                            console.log(error.message);
-                        });
+            await axios.post("/api/user/wishlist", {
+                "product": item
+            }, {
+                headers: {
+                    authorization: localStorage.getItem("feetz")
+                }
+            }).then((res) => {
+                console.log(res);
+                if (res.status === 201) {
+                    setWishListState(res.data.wishlist);
+                }
+            }).catch((error) => {
+                console.log(error.message);
+            });
+        } catch (err) {
+            console.log("error ", err.message);
+        }
+    }
+
+    const UpdateCartHandler = async (stateQuantity) => {
+        console.log(cartState.find((cartitem) => cartitem._id === _id));
+        let cartItem = cartState.find((cartitem) => cartitem._id === _id);
+        if (stateQuantity === 'increment' && (cartItem ?. qty < 4)) {
```

Comment:
Store this value in a variable (const)

Reply:
okay will create const variable and store them